Cargo.toml

```diff
@@ -61,7 +61,7 @@ openmls_x509_credential = { git = "https://github.com/wireapp/openmls", rev = "6
 proteus-traits = { git = "https://github.com/wireapp/proteus", tag = "v2.1.1" }
 proteus-wasm = { git = "https://github.com/wireapp/proteus", tag = "v2.1.1" }
 
-wire-e2e-identity = { git = "https://github.com/wireapp/rusty-jwt-tools", rev = "751f281ec75a6ae113584659b4b48940407c50f8", version = "0.9" }
+wire-e2e-identity = { git = "https://github.com/wireapp/rusty-jwt-tools", tag = "v0.11.0" }
```

Comment:
This changes from specifying a minor version to specifying a patch version. That's probably fine, but I don't know if there's a standard policy or convention for how specifically we specify our dependency versions.

Reply:
I think we follow the usual `version = ...` pattern for things published on crates.io, however none of our internal crates are published (e.g. rusty-jwt-tools, proteus-related things, openmls fork...). For those we just point to a tag, like above.